components/SubmissionsTable/index.spec.tsx

```diff
@@ -42,6 +43,16 @@ describe('SubmissionsTable', () => {
     expect(screen.getByText('No results to show.'));
   });
 
+  it('lets you search for a submission by form name', async () => {
+    render(<SubmissionsTable submissions={[mockSubmission]} />);
+
+    fireEvent.change(screen.getByRole('searchbox'), {
+      target: { value: 'Sandbox form' },
+    });
```

Comment:
Just to check I understand this: when we mock a submission in our tests, this submission is related to the form with an id of `foo` saved [here](https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/f8cb4b2db9a976c96307336b1dfddba1297c5cb4/data/flexibleForms/foo.ts) and that form has the name: `Sandbox form`, which is why we search for it in this test?

Reply:
exactly. were we to ever change that form this test would break